test/common/phantom-lib.js

```diff
@@ -8,11 +8,39 @@
 localStorage.clear();
 
 function ph_select(sel) {
-    var list, i, els = [];
-    if (window.jQuery)
+    var list, i, els = [], startIdx, braceCounter, searchText, containsSelector;
+    if (window.jQuery) {
         list = window.jQuery(sel);
-    else
-        list = document.querySelectorAll(sel);
+    } else {
+        containsSelector = ':contains(';
```

Comment:
I would suggest breaking this out into its own function. Other than that, this looks great.